### ISSUE TITLE
508-defect-2 [COGNITION]: Heading levels SHOULD increase by one in a logical order

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#16735]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

<hr/>

## Point of Contact

**VFS Point of Contact:**  Josh Kim

## User Story or Problem Statement

As a screen reader user, I want to navigate the hierarchy of the page content using heading levels to save time and frustration.

## Details

Headings on a web page should progress, increasing by one, in sections of content. There are a number of headings throughout the pages that don't adhere to a logical heading order of H1, H2, H3, H4, etc.

For example, it the page was written as an outline, it would not make sense for "How can I check the status of my application" to be at the same level as its child steps "1. Sign in to VA.gov..."

## Acceptance Criteria

Example:
- [x] Headings progress one by one in a logical order

## Environment

* Operating System: all
* Browser: all
* Screenreading device: any
* Server destination: staging and production

## Steps to Recreate

1. [Reach the submission complete page of the VA Form 28-8832](https://staging.va.gov/careers-employment/education-and-career-counseling/apply-career-guidance-form-28-8832/introduction).
2. Have developer tools open.
3. Inspect the subway map heading "How can I check the status of my application?"
4. Verify it is a h3.

## Solution (if known)

Update the headings into a logical order as demonstrated below. I have included visual heading classes to maintain the intended visual hierarchy.

### Before
```html
<h2 class="vads-u-font-size--h3 vads-u-margin-top--1">Thank you for submitting your application</h2>
  <h3 class="vads-u-font-size--h4">Education/Vocational Counseling Application <span class="additional">(Form 28-8832)</span></h3>
  <h3>What happens after I apply?</h3>
  <h3>How long will it take VA to process my application?</h3>
  <h3>How can I check the status of my application?</h3>
  <h3 class="vads-u-font-size--h5">Sign in to VA.gov</h3>
  <h3 class="vads-u-font-size--h5">If you haven't yet verified your identity, complete this process when prompted</h3>
  <h3 class="vads-u-font-size--h5">Go to your personalized My VA homepage</h3>
  <h3 class="vads-u-margin-top--1p5">What if I have more questions?</h3>
<h2 class="help-heading">Need help?</h2>
```

### After
```html
<h2 class="vads-u-font-size--h3 vads-u-margin-top--1">Thank you for submitting your application</h2>
  <h3 class="vads-u-font-size--h4">Education/Vocational Counseling Application <span class="additional">(Form 28-8832)</span></h3>
<h2 class="vads-u-font-size--h3">What happens after I apply?</h2>
<h2 class="vads-u-font-size--h3">How long will it take VA to process my application?</h2>
<h2 class="vads-u-font-size--h3">How can I check the status of my application?</h2>
  <h3 class="vads-u-font-size--h5">Sign in to VA.gov</h3>
  <h3 class="vads-u-font-size--h5">If you haven't yet verified your identity, complete this process when prompted</h3>
  <h3 class="vads-u-font-size--h5">Go to your personalized My VA homepage</h3>
  <h3 class="vads-u-margin-top--1p5">What if I have more questions?</h3>
<h2 class="help-heading">Need help?</h2>
```

## WCAG or Vendor Guidance (optional)

[Headings - Web Accessibility Tutorials](https://www.w3.org/WAI/tutorials/page-structure/headings/)

## Screenshots
<img width="1792" alt="Screen Shot 2020-12-16 at 6 13 01 PM" src="https://user-images.githubusercontent.com/14154792/102417989-dede8200-3fca-11eb-9828-9e3c1e9ff79f.png">

**Comment by jason-gcio** [comment]
We can do this.  We would like to know how to catch this in the future...question type lines are one header size, and I guess the answers are in the next one?

**Comment by joshkimux** [comment]
@jason-gcio you could consider using the [Chrome extension headingsMap](https://chrome.google.com/webstore/detail/headingsmap/flbjommegcjonpdmenkdiocclhjacmbi?hl=en) to quickly check the outline of the page based on headings. If items aren't ordered in a way that makes sense, you can rearrange the headings accordingly.

**Comment by 1Copenut** [comment]
@joshkimux If you have a chance would you mind adding a screen shot to this ticket? I think it'll help the dev team working on it to visualize the right hierarchy.

**Comment by joshkimux** [comment]
@1Copenut I've attached a screenshot accordingly

**Comment by jason-gcio** [comment]
@joshkimux - would you like to look at this before closing?

**Comment by joshkimux** [comment]
@jason-gcio It looks like this hasn't been merged in yet! Once it is merged, we should be good to close! 👍 
https://github.com/department-of-veterans-affairs/vets-website/pull/15505/files
